neo/IO/Data/RocksDB/RocksDBCore.cs

```diff
@@ -0,0 +1,181 @@
+using RocksDbSharp;
+using System;
+using System.Collections.Generic;
+
+namespace Neo.IO.Data.RocksDB
+{
+    public class RocksDBCore : IDisposable
+    {
+        public static readonly Options OptionsDefault = new Options();
+        public static readonly ReadOptions ReadDefault = new ReadOptions();
```

Comment:
Should we create a new `ReadOptions` every time?

Reply:
I think that is better reusing it, there are no problem at all

Reply:
We need to do a test:

```
Put(key1, value1);
Get(ReadDefault, key1);
Put(key2, value2);
Get(ReadDefault, key2); //We need to ensure that we can successfully read key2.
```

Because I am worried that the default snapshot will cause the second read to fail.